### PR TITLE
configurable scroll element

### PR DESCRIPTION
**Pull request by ktrhn**
Makes the scrolled element configurable. Should propably solve https://github.com/dollarshaveclub/ember-router-scroll/issues/56

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,15 @@
 ember install ember-router-scroll
 ```
 
+### Options
+You can specify the id of an element for which the scroll position is saved and set. Default is `window` for using the scroll position of the whole viewport. You can pass an options object in your applications `config/environment.js` file.
```
Comment (briangonzalez) [inline README.md]: > application's `config/environment.js`

File: addon/index.js [hunk 2]
```diff
@@ -28,13 +28,24 @@ export default Mixin.create({
...
     let scrollPosition = get(this, 'service.position');
 
     let preserveScrollPosition = transitions[transitions.length - 1]
       .handler.controller.get('preserveScrollPosition');
 
     if (!preserveScrollPosition) {
-      window.scrollTo(scrollPosition.x, scrollPosition.y);
+      if ('window' === scrollElement) {
+        window.scrollTo(scrollPosition.x, scrollPosition.y);
+
```
Comment (briangonzalez) [inline addon/index.js]: Remove line.

File: addon/services/router-scroll.js [hunk 3]
```diff
@@ -1,20 +1,42 @@
...
+    const scrollElement = get(this, 'scrollElement');
     const scrollMap = get(this, 'scrollMap');
     const key = get(this, 'key');
+    let x;
+    let y;
+
+    if ('window' === scrollElement) {
+      x = window.scrollX;
+      y = window.scrollY;
+
```
Comment (briangonzalez) [inline addon/services/router-scroll.js]: Remove line.

File: README.md (hunk 1 above)
Comment (briangonzalez) [inline README.md]: @ktrhn can you put in an apostrophe in the word "application"?

Comment (ktrhn) [inline README.md]: I have in the last commit: https://github.com/dollarshaveclub/ember-router-scroll/pull/67/commits/370bb90b28b628dbe0ba25ab6c549a480836bd73#diff-04c6e90faac2675aa89e2176d2eec7d8

Comment (briangonzalez) [inline README.md]: Awesome, thanks!

**Pull request merged**